views/pages/personal.njk

```diff
@@ -5,23 +5,24 @@
     
     <form action="{{ route('personal.store') }}" method="post">        
         {{ textInput('socialInsuranceNumber', 'section1.sin', { hint: 'section1.sin_example', class: "w-3-4", required: true }) }}
-
-        {{ radioButtons('preferredTitle', { '1':'section1.preferred_title_mr','2':'section1.preferred_title_mrs','3':'section1.preferred_title_miss','4':'section1.preferred_title_ms'}, data.preferredTitle, 'section1.preferred_title', errors) }}
         
         {{ textInput('firstName', 'section1.first_name', { required: true }) }}
         {{ textInput('middleName', 'section1.middle_name') }}
         {{ textInput('lastName', 'section1.last_names', { required: true }) }}
         {{ textInput('birthLastName', 'section1.birth_last_name') }}
 
-        {{ textInput('birthdate', 'section1.birthdate', { hint: 'year_month_day_format', required: true }) }}
+        {{ dateInput('birthdate', data.birthdate, __('section1.birthdate'), { required: true }) }}
 
-        {{ textArea('address', 'section1.mailing_address', { hint: 'section1.mailing_address_hint', required: true}) }}
+        {{ textInput('address', 'section1.mailing_address', { required: true}) }}
+        {{ textInput('city', 'section1.city', { required: true}) }}
+        {{ textInput('province', 'section1.province', { required: true}) }}
+        {{ textInput('country', 'section1.country', { required: true}) }}
+        {{ textInput('postal', 'section1.postal', { required: true}) }}
+        {{ textInput('email', 'section1.email', { hint: 'section1.email.hint', required: true}) }}
 
         {{ textInput('telephone', 'section1.telephone', { hint: 'telephone_format', required: true }) }}
```

Comment:
Instead of `Telephone number`, this can just be `Phone number`.
We don't have hint text in the original design – is any validation done on this field? We should try to take any format, if possible.

Reply:
removed the telephone format validation, updated the labels, removed hint text